raphtory-benchmark/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 
 [dependencies]
 criterion = { workspace = true }
-raphtory = { path = "../raphtory", features = ["io"], version = "0.11.3" }
+raphtory = { path = "../raphtory", features = ["io", "proto"], version = "0.11.3" }
```

Comment:
This means that anyone that doesn't have protoc will have the `cargo test` command fail